test/platform/github/index.spec.ts

```diff
@@ -4,6 +4,7 @@ import { GotApi } from '../../../lib/platform/common';
 describe('platform/github', () => {
   let github: typeof import('../../../lib/platform/github');
   let api: jest.Mocked<GotApi>;
+  let got: any;
```

Comment:
```suggestion
  let got: jest.Mock<Promise<GotResponse>>;
```
This would be the correct type (import `GotResponse` from `'../../../lib/util/got'`)

Reply:
Wow, special thanks for this, I missed right type for this one. But, instead of `GotResponse`, can we just use `{ body: object }` in order to avoid too much required fields?

Reply:
Better
```suggestion
  let got: jest.Mock<Promise<Partial<GotResponse>>>;
```